gnomad/resources/resource_utils.py

```diff
@@ -168,6 +147,8 @@ class PedigreeResource(BaseResource):
     :param missing: The string used to denote missing values. For case-control, 0, -9, and non-numeric are also treated as missing.
     """
 
+    expected_file_extensions: List[str] = [".fam", ".ped"]
+
     def __init__(
```

Comment:
Do you keep the __init__ function here because you need to define extra parameters unique to pedigrees (`quant_pheno`, 'delimiter`, 'missing')?

Reply:
Yes